ios/FluentUI/Vnext/List/List.swift

```diff
@@ -53,66 +55,64 @@ public struct MSFListView: View {
 
     public var body: some View {
         let sections = state.sections
-        List {
-            ForEach(sections, id: \.self) { section in
-                if let sectionTitle = section.title {
-                    if #available(iOS 14.0, *) {
-                        Section(header: Header(title: sectionTitle, tokens: tokens)) {}
-                            .textCase(.none)
-                            .listRowInsets(EdgeInsets())
-                            .padding(.top, tokens.horizontalCellPadding / 2)
-                            .padding(.leading, tokens.horizontalCellPadding)
-                            .padding(.trailing, tokens.horizontalCellPadding)
-                            .background(Color(tokens.backgroundColor))
-                    } else {
-                        Text(sectionTitle)
-                            .listRowInsets(EdgeInsets(top: tokens.horizontalCellPadding / 2,
-                                                      leading: tokens.horizontalCellPadding,
-                                                      bottom: tokens.horizontalCellPadding / 2,
-                                                      trailing: tokens.horizontalCellPadding))
-                            .font(Font(tokens.subtitleFont))
-                            .foregroundColor(Color(tokens.subtitleColor))
-                    }
-                }
+            ScrollView {
+                VStack(spacing: 0) {
+                    ForEach(sections, id: \.self) { section in
+                        if let sectionTitle = section.title {
+                            Header(title: sectionTitle, tokens: tokens)
+                        }
 
-                ForEach(section.cells, id: \.self) { cell in
-                    MSFListCellView(cell: cell,
-                                    tokens: tokens)
-                        .border(section.hasBorder ? Color(tokens.borderColor) : Color.clear, width: section.hasBorder ? tokens.borderSize : 0)
-                        .frame(maxWidth: .infinity)
+                        ForEach(section.cells.indices, id: \.self) { index in
+                            let cell = section.cells[index]
+                            let hasDividers = (index < section.cells.count - 1 && section.hasDividers) || (section.hasDividers && (cell.children != nil))
+                            MSFListCellView(cell: cell,
+                                            tokens: tokens,
+                                            hasDividers: hasDividers)
+                                .frame(maxWidth: .infinity)
+                        }
+                        Divider()
```

Comment:
Should this also respect the hasDividers property?

![Screen Shot 2021-01-25 at 11 44 28 AM](https://user-images.githubusercontent.com/68076145/105757326-afdcf980-5f02-11eb-93ef-1e4fd8d52719.png)

Reply:
Adjusted